edgelet/iotedge/src/init/import/mod.rs

```diff
@@ -268,7 +268,7 @@ fn execute_inner(
                     ),
             }) => (
                 aziot_identityd_config::Provisioning {
-                    dynamic_reprovisioning: *dynamic_reprovisioning,
+                    always_reprovision_on_startup: *dynamic_reprovisioning,
```

Comment:
No. `always_reprovision_on_startup` and `dynamic_reprovisioning` are not the same thing.

`always_reprovision_on_startup` needs to be set to what it was set in the old config.

Reply:
@massand said this is the fix :cry:

Reply:
I don't know why he said that, but it doesn't change what I said.

Reply:
Wait, so what should the fix be for the `ProvisioningType::Manual` cases?

Reply:
@arsing

Reply:
Also, on that note, it looks like I need to change the test-files as well.

I can't say I'm a fan of how these changes are getting bundled in with this unrelated PR, but if it's happening, I could use some guidance on what to do.

Reply:
https://github.com/Azure/iotedge/blob/release/1.0.10/edgelet/edgelet-core/src/settings.rs#L426-L430

Reply:
Gotcha. So I'll just set them to true then. Cool.

Reply:
It is done